src/tests/ftest/util/AgentUtils.py

```diff
@@ -111,11 +164,28 @@ def stop_agent(client_list):
 
     client_list -- kill daos_agent on these hosts
     """
-
     if len(sessions) == 0:
         return
 
     # this kills the agent
     for client in client_list:
         cmd = "pkill daos_agent"
-        resp = subprocess.call(["ssh", client, cmd])
\ No newline at end of file
+        resp = subprocess.call(["ssh", client, cmd])
```

Comment:
As the code is currently, I don't think you need `resp =` here do you?

```suggestion
        subprocess.call(["ssh", client, cmd])
```

However, that said, it is probably prudent to leave it as is and check the value of `resp` to see if any of the `ssh` (or `pkill`) commands fail and start the process of raising errors if that happens, or retry, etc.

Reply:
Unless I am misunderstanding this code, this is raising error if a process isn't killed. We pkill the process, check to see if the process still exists on node 5 seconds later, if it does, we raise AgentFailed to notify the user and continue.

Reply:
Yes, we do check for agents later, but if this `ssh` fails, this is the first sign that things have gone wrong and we should heed it here rather than trying to make up for it later.